### ISSUE TITLE
The response cannot be serialized normally.

### ISSUE THREAD

**Issue by fealone** [odd12258053/agraffe#6]
If return the response using `Dict` or `jsonable_encoder`, the following error will be sent.

```
[ERROR] Runtime.MarshalError: Unable to marshal response: b'{}' is not JSON serializable
```

Do I have the resolve method?

**Comment by odd12258053** [comment]
@fealone 
Thank you for your comment.
Now I released version 0.3.
In version 0.3 and above, agraffe supports the HTTP API.
Please update the library and one more try to requests.

**Comment by fealone** [comment]
Thank you for your quick support.
I was able to confirm that it works normally.
This issue closes.
